source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md

```diff
@@ -112,10 +118,72 @@ Encoded, this query would look like this:
 
 `status%3D%22bouced%22`
 
+{% anchor h2 %}
+Creating compound queries
+{% endanchor %}
+
+You may want to combine queries for a compound query. For example, you could filter for emails between a date range, or you could filter for when a specific recipients email is bounced. Here are some common use cases:
+
+{% anchor h3 %}
+Filter by a reciepeint email that was bounced
+{% endanchor %}
+
+Use this query to filter by a recipents email and by emails that are bounced: (replace `<<your API key>>` with an API key from your account, and replace <<email>> with the recepients email):
+
+{% codeblock %}
+curl --request GET \
+  --url 'http://api.sendgrid.com/v3/messages?limit=10&query=to_email%3D<<email>>+status%3D%22bouced%22' \
+  --header 'authorization: Bearer <<your API key>>'
+{% endcodeblock %}
+
+{% anchor h3 %}
+Filter by date range
+{% endanchor %}
+
+Use this query to filter to emails between specific dates: (replace `<<your API key>>` with an API key from your account, and replace {start_date} and {end_date} with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`)
+
+{% codeblock %}
+curl --request GET \
+  --url 'http://api.sendgrid.com/v3/messages?limit=10&query>{start_date}<{end_date}' \
+  --header 'authorization: Bearer <<your API key>>'
+{% endcodeblock %}
+
+{% anchor h3 %}
+Filter by a reciepient and and a date range
+{% endanchor %}
+
+Use this query to filter to emails by recepient and between specific dates: (replace `<<your API key>>` with an API key from your account, replace <<start_date>> and <<end_date>> with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`, and and replace <<email>> with the recepients email)
```

Comment:
+Use this query to filter to emails by recipient and between specific dates: (replace `<<your API key>>` with an API key from your account, replace <<start_date>> and <<end_date>> with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`, and and replace <<email>> with the recipient's email)